src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -28,6 +30,15 @@
 class DebugAutowiringCommand extends ContainerDebugCommand
 {
     protected static $defaultName = 'debug:autowiring';
+    private $supportsHref;
+    private $fileLinkFormatter;
+
+    public function __construct(FileLinkFormatter $fileLinkFormatter = null)
```

Comment:
this should take `string $name = null` as 1st argument to not break BC

Reply:
Fixed, thanks!

Reply:
@nicolas-grekas are you sure that I have to add `string $name = null` to the constructor? Since I didn't see any other Command class doing it also I've checked the BC Promise Policy from Symfony website and `@final` classes are an exception on the promise (and this class is marked with `@final`?

Reply:
:100: sure, see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Command/Command.php#L69

Reply:
Okay true, I've looked at "using our code" section, Sorry! But why other command classes do not have `string $name = null` in their constructor? Is this something that should be fixed too?

Reply:
well, if they never exposed the parent constructor in a release, they don't need to preserve BC with its usage.